website/static/css/timeline.css

```
@@ -0,0 +1,51 @@
+.timeline-container {
+  max-width: 800px;
+  margin: auto;
+}
+
+.timeline-event {
+  margin: 0;
+
+  position: relative;
+  display: flex;
+
+  text-align: left;
+}
+
+.timeline-event::before {
+  content: "";
+  background-color: #ffd533;
```

```suggestion
  background-color: #f5da55;
```
Can re-use the primary colour.